src/Stratis.Bitcoin/Controllers/Models/TransactionModel.cs

```diff
@@ -26,7 +26,7 @@ public TransactionModel(Transaction trx)
         }
 
         /// <summary>The hashed transaction.</summary>
```

Comment:
Can you please also fix this comment while this PR is open

Reply:
👍